docs/switching-to-v2.md

```diff
@@ -0,0 +1,35 @@
+# Switching to V2
+## Java distribution
+The major breaking change in V2 is the new mandatory input `distribution`. This field should be specified with one of supported distributions. See [Supported distributions](../README.md#Supported-distributions) section for the list of available options.  
```

Comment:
First sentence: `The major breaking change in V2 is the new mandatory "distribution" input.`

Also: `See [Supported distributions](../README.md#Supported-distributions) for a list of available options.`